app/components/TrySomethingCarousel/TrySomethingCarousel.tsx

```diff
@@ -0,0 +1,83 @@
+import _ from 'lodash';
+import React, { useCallback, useState } from 'react';
+import { View } from 'react-native';
+import ReanimatedCarousel from 'react-native-reanimated-carousel';
+import { GestureHandlerRootView } from 'react-native-gesture-handler';
+import { useSharedValue } from 'react-native-reanimated';
+
+import { WINDOW_WIDTH } from '../../constants/window';
+import { generalStyles } from '../../utils/styles';
+import { ITrySomethingCarouselProps } from './TrySomethingCarousel.types';
+import { styles } from './TrySomethingCarousel.styles';
+import { ProgressBarItem, TrySomethingCarouselItem } from './components';
+import {
+  CAROUSEL_MODE_CONFIG,
+  ITrySomethingCarouselItem,
+} from './TrySomethingCarousel.data';
+
+export const TrySomethingCarousel: React.FC<ITrySomethingCarouselProps> = ({
+  data,
+  setIndex,
+}) => {
+  const progressValue = useSharedValue(0);
+
+  const [currentPosition, setCurrentPosition] = useState(0);
+
+  const onSnapToItem = useCallback(
+    index => {
+      setIndex(Math.floor(index));
+    },
```

Comment:
https://stackoverflow.com/questions/35590543/how-do-you-chain-functions-using-lodash

Reply:
```_.flow(
 yourFunction1,
 yourFunction2,
)(foundUser);

Reply:
Play around it please